harness/helpers.ts

```diff
@@ -118,17 +119,36 @@ export const wait = (ms: number) => {
     })
 }
 
-export const callOnClientInitialized = async (clientId: string, url: string, callbackURL: string) => {
-    return await fetch(`${url}/client/${clientId}`, {
+export const callOnClientInitialized = async (
+    clientId: string, 
+    proxyUrl: string, 
+    mockServerUrl: string,
+    scope: Scope
+) => {
+    let resolveIsInitialized: (value: unknown) => void
+
+    const isInitialized = new Promise(function(resolve){
```

Comment:
might want to add a timeout to this so it doesn't hang indefinitely if never called